store/tikv/gcworker/gc_worker.go

```diff
@@ -235,13 +235,11 @@ func (w *GCWorker) leaderTick(ctx context.Context) error {
 		if err != nil {
 			metrics.GCJobFailureCounter.WithLabelValues("prepare").Inc()
 		}
-		w.gcIsRunning = false
```

Comment:
I think removing it will cause `gcIsRunning` cannot flip to `false` any more.

Reply:
If i'm right, here w.gcIsRunning must be false because of the check at `L227`. The only situation of changing it to `false` is at L179, and `w.done` is sent at L262. Please double check.

Reply:
oh, you are right. I made a mistake.

Reply:
Does other routine may change the `w.gcIsRunning`?

Reply:
No